tests/requirement/links/test_requirement_links.py

```
@@ -0,0 +1,80 @@
+
+from django.test import TestCase
+from logging import getLogger
+import shutil
```

Ruff (0.11.9)

`shutil` imported but unused

Remove unused import: `shutil`

(F401)